storybook/stories/chart/piechart/PieChart.stories.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import { Pie, PieChart, ResponsiveContainer } from '../../../../src';
+import { pageData } from '../../data';
+
+export default {
+  title: 'Recharts/Chart/Pie Chart',
+  component: PieChart,
+};
+
+export const Simple = {
+  render: (args: Record<string, any>) => {
+    return (
+      <ResponsiveContainer width="100%" height={400}>
+        <PieChart width={800} height={400} startAngle={180} endAngle={0} outerRadius={80}>
```

Comment:
```suggestion
        <PieChart {...args}>
```
Lets spread the args to make the controls functional.